charts/incubator/alfresco-api-gateway/templates/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "alfresco-api-gateway.fullname" . }}
+  labels:
+    app: {{ template "alfresco-api-gateway.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "alfresco-api-gateway.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      imagePullSecrets:
+        - name: {{ .Values.registryPullSecrets }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - containerPort: {{ .Values.service.internalPort }}
+          env:
+            - name: ACT_KEYCLOAK_URL
+              value: "{{required "A valid keycloakURL needs to be set" .Values.keycloakURL }}"
+            - name: ACT_EUREKA_URL
+              value: "{{ .Values.eurekaURL | default (printf "http://%s-alfresco-activiti-cloud-registry:8761/eureka/" .Release.Name) }}"
+            - name: ACT_RABBITMQ_HOST
+              value: "{{.Values.rabbitmqReleaseName }}"
```

Comment:
Can we do the same thing here and for gateway as the above EUREKA URL in a subsequent pull request, a default value pulled from release name?

Reply:
Yes.  I'll create another PR for this.